reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+---
+ms.date:  2017-12-04
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the program engine and starting and stopping PowerShell
```

Comment:
I don't think the word `program` is necessary here.  Also, I think we can just say `providers` rather than `PowerShell providers` otherwise we would put `PowerShell` in front of everything.

Reply:
fixed